src/OpenTelemetry/Metrics/Builder/MeterProviderBuilderSdk.cs

```diff
@@ -15,8 +15,8 @@ namespace OpenTelemetry.Metrics;
 /// </summary>
 internal sealed class MeterProviderBuilderSdk : MeterProviderBuilder, IMeterProviderBuilder
 {
-    public const int MaxMetricsDefault = 1000;
-    public const int MaxMetricPointsPerMetricDefault = 2000;
+    public const int DefaultMaxMetricStreams = 1000;
```

Comment:
Is this "metric stream"? (I think it is "metric")

Reply:
I left the public API alone (we can mess with that in a follow-up if needed) but I renamed the internal concept from "maxMetricStreams" to "metricLimit" how you feel about that?

Reply:
Sounds good to me 👍